move_group:
  ros__parameters:
    moveit_simple_controller_manager:
      controller_names:
        - dual_arm_joint_trajectory_controller

      dual_arm_joint_trajectory_controller:
        action_ns: follow_joint_trajectory
        type: FollowJointTrajectory
        default: true
        joints:
          - left_joint_1
          - left_joint_2
          - left_joint_3
          - left_joint_4
          - left_joint_5
          - left_joint_6
          - right_joint_1
          - right_joint_2
          - right_joint_3
          - right_joint_4
          - right_joint_5
          - right_joint_6

